### PR TITLE
Poc/react native

### PR DESCRIPTION
**Pull request by Karvlox**
Clickup: https://app.clickup.com/t/868a7pgt4

### Reviewers 
@JP-Arequipa 
@CatCoder36

### CODE REVIEW

**Review by CatCoder36 — APPROVED** [review]

**Review by JP-Arequipa — APPROVED** [review]

File: apps/CEventService.API/obj/CEventService.API.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (JP-Arequipa) [inline apps/CEventService.API/obj/CEventService.API.csproj.nuget.dgspec.json]: remove this obj folder from the PR

File: apps/CEventService.API/obj/Debug/net8.0/CEventService.API.AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+//------------------------------------------------------------------------------
```
Comment (JP-Arequipa) [inline apps/CEventService.API/obj/Debug/net8.0/CEventService.API.AssemblyInfo.cs:1]: remove this Debug/net8.0 folder from the PR

File: Solidarios.sln [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿
```
Comment (JP-Arequipa) [inline Solidarios.sln:1]: this is a config file of visual studio, it shouldn't be included in the repo

File: apps/CEventService.API/obj/CEventService.API.csproj.nuget.dgspec.json (hunk 1 above)
Comment (Karvlox) [reply apps/CEventService.API/obj/CEventService.API.csproj.nuget.dgspec.json]: Those files do not correspond to my files, it is as if I have pulled files from another branch.

File: apps/CEventService.API/obj/Debug/net8.0/CEventService.API.AssemblyInfo.cs (hunk 2 above)
Comment (Karvlox) [reply apps/CEventService.API/obj/Debug/net8.0/CEventService.API.AssemblyInfo.cs:1]: Those files do not correspond to my files, it is as if I have pulled files from another branch.

**Pull request closed without merging**